backend/prompt_studio/prompt_studio/views.py

```diff
@@ -41,3 +47,94 @@ def get_queryset(self) -> Optional[QuerySet]:
         else:
             queryset = ToolStudioPrompt.objects.all()
         return queryset
+
+    @action(detail=True, methods=["post"])
+    def reorder_prompts(self, request: HttpRequest) -> Response:
```

Comment:
@tahierhussain [MINOR] It is better to use `Request` object of `rest_framework` to maintain consistency through out the project and for easy parsing. [Ref](https://www.django-rest-framework.org/api-guide/requests/)

Reply:
@harini-venkataraman Thanks for the suggestion. Done.